ethereum_history_api/oracles/README.md

```diff
@@ -1,3 +1,31 @@
 ## Noir Ethereum history api oracles
 
 TypeScript oracle server that provides data for circuits.
+
+### Preparing fixtures
+
+Fixtures are data used for testing. They are downloaded and kept as local data. There are two types of fixtures: noir and js. They are pepared in different format and are used in different tests.
```

Comment:
I don't think this paragraph is needed

Reply:
What are fixtures is a general knowledge. The fact that we have two types is important, but `different format and are used in different tests` doesn't give any information.

Maybe something like:
### Noir Fixtures
Noir fixtures are generated as noir modules and consumed in Noir tests
### JavaScript Fixtures
JavaScript fixtures are JSON files which cache JSON RPC request/responses pairs
JavaScript fixtures are consumed either directly in unit tests or through a Mock client in integration tests